model/model/v1alpha/model.proto

```diff
@@ -510,16 +504,20 @@ message TriggerNamespaceLatestModelRequest {
   string namespace_id = 1 [(google.api.field_behavior) = REQUIRED];
   // Model ID
   string model_id = 2 [(google.api.field_behavior) = REQUIRED];
-  // Inference input parameters.
-  repeated TaskInput task_inputs = 3 [(google.api.field_behavior) = REQUIRED];
+  // Deleteted field.
+  reserved 3;
+  // Model inference inputs.
+  repeated google.protobuf.Struct task_inputs = 4 [(google.api.field_behavior) = REQUIRED];
 }
 
 // TriggerNamespaceLatestModelResponse contains the model inference results.
 message TriggerNamespaceLatestModelResponse {
   // Task type.
   common.task.v1alpha.Task task = 1;
+  // Deleted field.
+  reserved 2;
   // Model inference outputs.
-  repeated TaskOutput task_outputs = 2;
+  repeated google.protobuf.Struct task_outputs = 3;
```

Comment:
We adopted the convention of defining a `field_behavior` in every field 🙏. We should put this in the contribution guidelines soon.

Reply:
ohh cool, let me add those.